review port documentation; add EACCES as an acceptable error (#1363)
observablehq/framework: docs/getting-started.md

↳ <u><a href="http://127.0.0.1:3000/" style="color: inherit;">http://127.0.0.1:3000/</a></u></pre>

<div class="note">

If port 3000 is in use, the preview server will choose the next available port, so your actual port may vary. To specify a different port, use <nobr>`--port`</nobr>. For example, to run the preview server with npm on port 4321:

> > `npm run dev -- --port 4321`

Reviewer: ````suggestion
```sh
npm run dev -- --port 4321
```
````

For security, the preview server is by default only accessible on your local machine using the [loopback address](https://en.wikipedia.org/wiki/Localhost) 127.0.0.1. To allow remote connections, use <nobr>`--host 0.0.0.0`</nobr>.

> Port numbers below 1024 are usually protected and might need admin priviledges. For example, to share your preview server on the default http port:

Reviewer: ```suggestion
Port numbers below 1024 may need admin privileges. For example, to share your preview server on the default HTTP port 80:
```

> > `sudo yarn dev --host 0.0.0.0 --port 80`

Reviewer: ```suggestion
> `sudo npm run dev -- --host 0.0.0.0 --port 80`
```

I think we should bias towards npm rather than Yarn and be consistent if we’re only showing one.
